lib/whenever/numeric_seconds.rb

```diff
@@ -1,17 +1,11 @@
 module Whenever
   class NumericSeconds
-    PATTERN = /(\d+)\.(seconds?|minutes?|hours?|days?|weeks?|months?|years?)/
-
     attr_reader :number
 
     def self.seconds(number, units)
       new(number).send(units)
     end
 
-    def self.process_string(string)
-      string.gsub(PATTERN) { Whenever.seconds($1, $2) }
-    end
-
     def initialize(number)
       @number = number.to_i
```

Comment:
@javan reason I chose monkey patch Integer was this line. So, I didn't expect like `1.5.days`